### DISCUSSION TITLE
How to check what version I'm running

### DISCUSSION THREAD

**Discussion by sunilnc** [dani-garcia/vaultwarden#5288] [Q&A] [answered]
Hello all,

Is there a way to check what version of Vaultwarden I'm running?

For example, I am aware that I can see what version here:
![1](https://github.com/user-attachments/assets/63648a8e-cc1d-4dfe-bd80-bda97fe20324)

But how do I correlate it with this to always check if I have the latest version?:
![2](https://github.com/user-attachments/assets/2ecaf356-bdfc-4d90-ac15-fe596c5b924c)

**Comment by BlackDex** [comment, marked as answer]
Those versions are not the same. The one on the web-vault is the version of the web-vault, not of Vaultwarden.

There are multiple ways to do it. But the most efficient one is by checking it via the Vaultwarden Backend Admin `/admin/diagnostics`. That page will also show you if there are any configuration errors.

You can also check `https://sub.domain.tld/api/version`
Or, use the CLI
```bash
# Running it directly if installed locally
./vaultwarden --version

# Via docker
docker exec -it vaultwarden /vaultwarden --version
```

**Reply by sunilnc** [reply]
Thank you, this worked for me. Appreciate the help.
